hapijs/hapijs.com
# Fix “Community” section’s image alignment bug

* **Problem:** `section` and `.cf` styles both modify their `:after` pseudo-selector styles in a manner that collides (see #458).
* **Solution:**
    * Separate `section` and bottom rule appearance by creating a `.bottom-rule` utility class in the _public/css/components/base.styl_ partial stylesheet. This differentiates content – with meaning in a document – from its appearance.
    * Add the new `.bottom-rule` class to all `section`s in _templates_, except the last company `section` in _public/css/sections/community.styl_
    * Separate all elements that have both `.cf` and `.bottom-rule` classes. These are not compatible, so a `div.cf` is created as a child of `.bottom-rule`.
* **Testing:**
    1. Spin up a development of hapijs.com
    2. Observe consistent styles between https://hapijs.com and the development version on all `section` touchpoints, especially:


        ```shell
        $ git diff master..bug/#458-community-image-alignment --name-only -- templates/
        templates/community.jade
        templates/contribute.jade
        templates/error.jade
        templates/hapidays.jade
        templates/index.jade
        templates/updates.jade
        ```
    3. Observe the Walmart logo is now correctly aligned on the _Community_ section:

    ![heyo-walmart](https://user-images.githubusercontent.com/1858316/29543105-a28c6432-8693-11e7-84b3-68e1291241b9.jpg)
